src/store/validation/validationHelpers.ts

```diff
@@ -0,0 +1,127 @@
+import {IBlock} from '@floip/flow-runner'
```

Comment:
Thanks for moving these out, I'm think this will let us to have cleaner `validation` module 🙏

Reply:
Your welcome :)